Makefile

```diff
@@ -34,8 +34,7 @@ ifeq ($(BUILD_DEBUG), 1)
 	GO_OPT+= -gcflags="all=-N -l"
 endif
 
-GOTEST_OPT?= -race -timeout 30m -count=1
-GOTEST_OPT_WITH_COVERAGE = $(GOTEST_OPT) -cover
+GOTEST_OPT?= -race -timeout 30m -count=1 -cover
```

Comment:
Do we want to include `-v` here?  I'm good to drop it and bring it back if we miss it.

Reply:
I considered it, but left it out since we weren't doing it before.